src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -76,6 +77,15 @@ public void testInviteUser() throws IOException {
         // assertTrue(org.hasMember(user));
     }
 
+    @Test
+    public void testListMembersWithRole() throws IOException {
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+
+        List<GHUser> admins = org.listMembersWithRole("admin").asList();
```

Comment:
The bug was for `listMembersWithFilter`.  Can you add a test for that as well?

Reply:
Done!